Source/WTF/wtf/BloomFilter.h

```diff
@@ -105,8 +104,8 @@ inline std::pair<unsigned, unsigned> BloomFilter<keyBits>::keysFromHash(const st
     // We could use larger k value than 2 for long hashes.
     static_assert(hashSize >= 2 * sizeof(unsigned), "Hash array too short");
     return {
-        *reinterpret_cast_ptr<const unsigned*>(hash.data()),
-        *reinterpret_cast_ptr<const unsigned*>(hash.data() + sizeof(unsigned))
+        reinterpretCastSpanStartTo<unsigned>(std::span { hash }),
+        reinterpretCastSpanStartTo<unsigned>(std::span { hash }.subspan(sizeof(unsigned)))
```

Comment:
I would have put the span into a local instead of repeating the cast twice, even though the old code repeated the pointer expression twice.